docs/Technical-Documentation/TDP-environments-README.md

```diff
@@ -0,0 +1,62 @@
+# TDP Environments
+
+## Development
+
+| Dev Site | Frontend URL | Backend URL | Purpose |
+| -------- | -------- | -------- | -------- |
+| Sandbox     | https://tdp-frontend-sandbox.app.cloud.gov | https://tdp-backend-sandbox.app.cloud.gov     | Space for devs to test in a deployed environment 
+| A11y | https://tdp-frontend-a11y.app.cloud.gov | https://tdp-backend-a11y.app.cloud.gov | Space for accessibility testing |
+| QASP | https://tdp-frontend-qasp.app.cloud.gov | https://tdp-backend-qasp.app.cloud.gov | Space for QASP review |
+| raft | https://tdp-frontend-raft.app.cloud.gov | https://tdp-backend-raft.app.cloud.gov | Space for raft review |
+
+### Common Dependencies
+
+**Cloud.gov AWS RDS `(tanf-dev)`**
+- `tdp-db-dev`
+  
+**Cloud.gov AWS S3 `(tanf-dev)`**
+- `tdp-staticfiles-dev`
+- `tdp-datafiles-dev`
+- `tdp-tf-states` - Stores the Terraform state files used to create and re-recreate services infrastructure.
+
```

Comment:
I would add a dependency for clamav-rest to each of the sections here as well as in the diagram. There will be one instance of ClamAV per cloud.gov space.